### ISSUE TITLE
Implement vocab parall crossentropy loss

### ISSUE THREAD

**Issue by loopinf** [EleutherAI/oslo#79]
## Describe a TODO feature

- Add vocab cross entropy loss

## Assignees

-
